UM/Controller.py

```diff
@@ -265,7 +266,7 @@ def setActiveTool(self, tool: Optional[Union["Tool", str]]):
         from UM.Scene.Selection import Selection  # Imported here to prevent a circular dependency.
         if not self._active_tool and Selection.getCount() > 0:  # If something is selected, a tool must always be active.
             if "TranslateTool" in self._tools:
-                self._active_tool = self._tools["TranslateTool"]  # Then default to the translation tool.
+                self._active_tool = self._tools[self._fallback_tool]  # Then default to the translation tool.
```

Comment:
Shouldn't you also change the `if "TranslateTool" in self._tools:` above?

Reply:
Great, thanks. Overlooked this one :smile: